unity-client/Assets/Scripts/MainScripts/DCL/Components/BuilderInWorld/DCLName.cs

```diff
@@ -0,0 +1,34 @@
+using DCL;
+using DCL.Components;
+using DCL.Controllers;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DCLName : BaseDisposable
+{
+    [System.Serializable]
+    public class Model
+    {
+        public string value;
+    }
+
+    public Model model;
+
+    public DCLName(ParcelScene scene) : base(scene)
+    {
+        model = new Model();
+    }
+
+    public override IEnumerator ApplyChanges(string newJson)
+    {
+        Model newModel = SceneController.i.SafeFromJson<Model>(newJson);
+        if(newModel.value != model.value)
+        {
+            model = newModel;
+            RaiseOnAppliedChanges();
+        }
+    
+        yield return null;
```

Comment:
```suggestion
        return null;
```
We don't need to skip a frame here, the `return null;` is properly handled from outside to cover this case and don't skip a frame after this coroutine execution when null is returned.